src/main/java/com/amazon/ion/impl/_Private_UtilsTrampoline.kt

```diff
@@ -0,0 +1,11 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.amazon.ion.impl
+
+import com.amazon.ion.SymbolToken
+
+// Exposes "package-private" java functions in _Private_Utils as "internal" kotlin functions
```

Comment:
I'm confused by this comment. The relevant methods in _Private_Utils are public.

Reply:
Good catch. It's `SymbolTokenImpl` that is package private, and it is exposed by the methods in `_Private_Utils`.